components/html-block/html-block.js

```diff
@@ -9,7 +9,13 @@ export const htmlBlockContentStyles = css`
 	.d2l-html-block-compact {
 		font-size: 0.8rem;
 		font-weight: 400;
-		line-height: 1.2rem;
+		line-height: 1.2;
```

Comment:
I noticed this while I was in here. Using a fixed default line-height here is extremely problematic because font sizes can be significantly larger than this. I'm unsure whether moving to a unitless ratio is actually ideal though. Would certainly entertain any thoughts on this.

Reply:
Isn't this now going to be 1.2X the size of the current font size in the context of where the blog is being rendered? For example, with `<div style="font-size:1000px"><d2l-html-block></d2l-html-block></div>` won't that now have a line height of `1200px`? vs. before it was based on the root font size.

Reply:
Yeah, that's why I'm unsure what the right answer is here. The previous value of `1.2rem` is problematic because in the case that we have that div, the line height gets pinned to `1.2 * rootFontSize` but still tries to fit a 1000px font in that line. So the lines overlap and things get messy. Basically I'm not sure fixing a line height makes a ton of sense when we're rendering user-authored HTML.

Reply:
e.g. At a 42px font size this is what it looks like now in main:
![HtmlBlockFontSize](https://user-images.githubusercontent.com/19732724/163822338-2bc108e9-aaa4-4a31-a9f3-1e74bd853ae4.PNG)

Reply:
Ah I see, so the flexible line height is more about making things look right in general and less about this particular "get rid of margins" feature. I'm curious... if you remove the `line-height` altogether, what's the inherited value from the outer page?

Reply:
Oh, sorry, yes. I conflated these things in my head because on the editor-side I had to make an equivalent change to avoid pop-in when the editor loads. But yeah, this is a separate issue for sure. I'll check what the default is.

Reply:
Interesting. On the demo page, nothing is explicitly inherited, which should default it to the initial value of `normal`... But that doesn't seem to happen and everything still overlaps, which isn't what I expected. I'll check the LMS.

Reply:
The LMS behaves similarly. Setting `line-height: inherit` explicitly has the same behaviour (different from `line-height: normal`), so despite Chrome being absolutely convinced that we're not setting a line-height anywhere, it doesn't seem to be defaulting properly. 😕

Reply:
Interesting. And I think in some places in the LMS we're including a reset stylesheet which may mess with the defaults even further.

Reply:
Yeah, this has made me extra confused about what behaviour is desired/expected in this case. A unitless line-height made sense to my frazzled end-of-week brain because it would maintain existing behaviour if `fontSize === rootFontSize` while also maintaining a consistent ratio of whitespace to font size... But this behaviour now makes me uncertain.

Reply:
Grant, how is that working above in the screenshot with `42px` font-size and `1.2rem` line-height?  The line-height would equate to `24px`... I would expect things to be overlapping a lot more.  🤔

Reply:
Oh yeah, it's much worse in the compact case. The screenshot above isn't from the compact case.